contracts/0.4.24/Lido.sol

```diff
@@ -123,9 +123,9 @@ contract Lido is ILido, IsContract, Pausable, AragonApp {
     /**
     * @notice Send funds to the pool with optional _referral parameter
     * @dev This function is alternative way to submit funds. Supports optional referral address.
-    * @return StETH Amount of StETH tokens generated
+    * @return Amount of StETH shares generated
```

Comment:
Explain why shares instead of tokens. We discussed this verbally, but this should be stated explicitly in commit message.

Reply:
Done